docs/admin/region-availability.md

```diff
@@ -11,13 +11,16 @@ ms.topic: overview
 ms.date: 08/30/2024
 ---
 
-# Fabric region availability
+# [Microsoft Fabric F SKU](../get-started/microsoft-fabric-overview.md) Regional Availability
 
-Microsoft Fabric is available in the Azure regions listed in this article. If your Microsoft Fabric home region isn't listed, you can still use Fabric by creating a Fabric capacity in a region where Fabric is available, and then using Fabric's muti-geo capabilities. For more information, see [Multi-Geo support for Fabric](./service-admin-premium-multi-geo.md).
+Microsoft Fabric F SKUs are available in the [Azure public cloud regions](https://azure.microsoft.com/en-us/explore/global-infrastructure/geographies/) listed in this article. As Azure launches new regions, or data centers become constrained, there may be a period when Power BI is the only or one of a few Fabric experiences offered on F SKUs. This article describes what is available in each region and is divided into three sections:
+1)	Azure regions with all Fabric experiences available
+2)	Azure regions that have a limited number of Microsoft Fabric experiences but include Power BI 
+3)	Azure regions that have a Fabric Public Preview experiences available
 
-To find out what your Fabric home region is, see [Find your Fabric home region](./find-fabric-home-region.md).
+There is a home region associated with your tenant ([Find your Fabric home region](./find-fabric-home-region.md)), if your home region isn’t listed below you won’t be able to access certain functionalities in that region. In such cases, you might want to create a capacity in a region where Fabric is available. For more information, see [Multi-Geo support for Fabric](./service-admin-premium-multi-geo.md).
 
-For more information about purchasing a Fabric subscription, see [Buy a Microsoft Fabric subscription](../enterprise/buy-subscription.md).
+For details about purchasing a Fabric subscription, see [Buy a Microsoft Fabric subscription](../enterprise/buy-subscription.md). Microsoft Fabric refers to the items grouped by experiences found [here](https://learn.microsoft.com/en-us/rest/api/fabric/articles/item-management/item-management-overview). These Microsoft Fabric items are available for users in the regions listed below:
```

Comment:
Move this to the overview. Also, I don't understand this sentence: 

Microsoft Fabric refers to the items grouped by experiences found [here](https://learn.microsoft.com/en-us/rest/api/fabric/articles/item-management/item-management-overview). These Microsoft Fabric items are available for users in the regions listed below:

Perhaps the link is wrong..?

Reply:
That link has the items listed by experience which explains how we are organizing the tables. If all items are available then the region is in the first list. If only a subset of items are available then the region goes in the second list. Then if preview items are available the region is designated in the third group of tables.